admin/zabbix/Makefile

```diff
@@ -29,7 +31,7 @@ define Package/zabbix/Default
   SECTION:=admin
   CATEGORY:=Administration
   TITLE:=Zabbix
-  URL:=http://www.zabbix.com/
+  URL:=https://www.zabbix.com/
```

Comment:
all the changes above are independet and should just go in separately.

Reply:
Done